packages/lisk-transactions/src/transactions/0_transfer.ts

```diff
@@ -78,7 +77,7 @@ export const transferAssetFormatSchema = {
 	},
 };
 
-const validateInputs = ({
+export const validateInputs = ({
```

Comment:
mentioned above, let's move this to `transfer.ts` above

Reply:
Done. Resolving.